src/modules/Cache.js

```diff
@@ -1,13 +1,25 @@
 import { getCurrentUrl, normalizeUrl } from '../helpers.js';
 
 export class Cache {
-	constructor() {
+	constructor(options) {
 		this.pages = {};
 		this.last = null;
+
+		const defaults = {
+			resolvePath: path => path
+		}
+		this.options = {
```

Comment:
Does spreading cause an error if a user passes `true` or `false` for the `cache` option? I'd rather we add a check here for object types.

Reply:
both `{...true}` as well as `{...false}` yields an empty object, so we should be fine. The cache doesn't really need to know about if it is enabled or not. Still, maybe it would be clearer upon inspection to resolve the enabled state into the options. I will take care of that.